content/css/jenkins.css

```diff
@@ -1426,3 +1426,26 @@ p.details {
 .imageblock.right {
   float: right;
 }
+
+
+#sponsorsblock {
+  padding: 1rem 2rem;
+}
+
+#sponsorsblock ul, p {
+  list-style-type: none;
+  padding: 0;
+  text-align: center;
```

Comment:
@rtyler adding "p" tag simply makes all the paragraph of the site center aligned.